es/tabs/manifiesto.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: manifesto
+# multilingual page pair id, this must pair with translations of this page. (This name must be unique)
+lng_pair: id_manifesto
+---
+<div class="padding-container even-background">
+    <div>
+        <div class="photoTitle">
+            <img class="imgTitles" src="/assets/img/default/logo.webp" alt="Logo image">
+            <h2 class="title2">Un enfoque colaborativo para hacer que la cultura sea más amable</h2>
+        </div>
+        <p>CuraCultura no es sólo una empresa. También es una filosofía, definida por su propio nombre:
+          cuidamos de las personas a través de la cultura, y cuidamos la cultura por el bien de las
+          personas. Nos propusimos hacer del sector cultural un lugar más amable para todos los que lo
+          habitan, ya sean inquilinos permanentes o visitantes poco frecuentes, trabajadores y
```

Comment:
"inquilinos" here sounds weird for me, maybe this should be reviewed or asked to the client